client/src/c_console.cpp

```diff
@@ -418,6 +448,84 @@ void ConsoleCommandLine::deleteCharacter()
 	}
 }
 
+
+void ConsoleCommandLine::deleteLeftWord()
+{
+	size_t word = 0;
+	bool firstSpacesCleared = false;
+	bool spaceWord = false;
+
+	const char* str = text.c_str();
+
+	// Delete trailing spaces instead of the word
+	if (str[cursor_position - 1] == ' ')
```

Comment:
Again, what happens if `cursor_position` is 0 here?

Reply:
Changed to use `isspace()` and check `cursor_position` for all of these.